src/uwtools/tests/drivers/test_driver.py

```diff
@@ -235,7 +236,7 @@ def test_Assets__rundir(assetsobj):
     assert assetsobj._rundir == Path(assetsobj._driver_config["rundir"])
 
 
-def test_Assets__validate(assetsobj):
+def test_Assets__validate_internal(assetsobj):
```

Comment:
Test name updated from `_validate` to `validate_external`. Although this test case didn't change maybe updating the test name to remove this extra underscore would be appropriate. 
 test_Assets__validate_internal to  test_Assets_validate_internal

Reply:
I think this new name is accurate: This test exists to test the `_validate()` method (see line 242). It's only incidental that it's interacting with the `validate_internal()` method, which we're mocking here. It's the behavior of `_validate()` -- namely, that it calls `validate_internal()` with the expected/correct arguments -- that this test is interested in.

I think I see where the name confusion might arise, though. This test could be described as
> Test `_validate()` with an internal schema.

which is not to be confused with a test of the `validate_internal()` function imported into this module. Would a more verbose name like `test_Assets__validate_with_internal_schema()` (and, similarly, below, `test_Assets__validate_with_external_schema()` help?